frontend/components/text-classifier/sidebar/TextClassificationRules.vue

```diff
@@ -124,7 +124,7 @@ export default {
     labels() {
       const labelsInRules =
         this.dataset.rules.flatMap((rule) => rule.labels) || [];
-      const availableLabels = this.dataset._labels.map((label) => {
+      const availableLabels = this.dataset.labels.map((label) => {
```

Comment:
Is this a bug? I mean, we should use the `dataset.labels` property always (even prior to this new feature)

Reply:
Yes, in this case `dataset._labels `was wrong